src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchRules.groovy

```diff
@@ -42,4 +44,20 @@ class SwitchRules {
     static Set<Long> missingRuleCookieIds(Collection<PathDiscrepancyDto> missingRules) {
         return missingRules.collect {new Long((it.getRule() =~ COOKIE_ID_IN_RULE_DISCREPANCY_STRING_REGEX)[0])}
     }
+
+    FlowEntry pingRule(String encapsulationType) {
+        def pingRuleCookie = getPingRuleCookie(encapsulationType)
+        return northboundService.getSwitchRules(switchId).flowEntries
```

Comment:
It would be nice to leave here todo with link to github issue where we are going to add HA-Flow id's to switch rules (IIRC, we planned something like this). It could be not obvious in future that this method can pick wrong rules if there're several HA-Flows on switch.

Reply:
Switch has 2 ping rules at most. One for transit_vlan flows and one for VXLAN flows. We do not need to know if it is a flow or Y flow or HA flow to get ping rules. We just need to know encapsulation type